### ISSUE TITLE
Could you add common colorbar to a panel plot script?

### ISSUE THREAD

**Issue by krishnaap** [haibuihoang/higrads#1]
It's possible to add a common color bar to a panel plot using vpage and tweak in cbar.gs, but can you add this feature to your script? It would be so helpful.

**Comment by haibuihoang** [comment]
You are right, currently h_cbarn only has the same size as one of the subplots. It's possible to add another cbar function with more flexible control based on h_cbarn.gsf. Because recently I am not working much with grads anymore, it will take some time before I add such function. If you can write it before, I can add it into the library.
